### PR TITLE
Moar documentation

### PR DESCRIPTION
**Pull request by Mwthorn**
Filling TODO's and change a few "symmetric" stuff

### CODE REVIEW

File: plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 1]
```diff
@@ -2659,14 +2663,27 @@ public void registerCoreMembers() {
         // @Group world
         //
         // @Description
-        // TODO: Document Command Details
+        // Specifying a player or list of players will only play
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: This is the end of a description, the random facts portion, and not a description in itself.

File: plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 2]
```diff
@@ -2659,14 +2663,27 @@ public void registerCoreMembers() {
...
         // @Usage
-        // TODO: Document Command Details
+        // Use to play a sound for a player
+        // - playsound <player> sound:ORB_PICKUP pitch:1
+        // @Usage
+        // Use to play a sound at a location for all nearby
+        // - playsound <player.location> sound:LEVEL_UP
+        // @Usage
+        // Use to notify all players with a sound
+        // - playsound <server.list_online_players> sound:LEVEL_UP volume:0.5 pitch:0.8
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: neither of the specified sounds exist in 1.10

File: plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java [hunk 3]
```diff
@@ -3801,7 +3818,20 @@ public void registerCoreMembers() {
         // <e@entity.location>
         //
         // @Usage
-        // TODO: Document Command Details
+        // Use to teleport a player to the its cursor is pointing on
```
Comment (mcmonkey4eva) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: you accidentally a word

Comment (Mwthorn) [inline plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java]: whoops! my bad.

**Pull request merged**